requirements.txt

```
@@ -3,11 +3,11 @@ click>=6.0,<=6.99
 funcsigs==1.0.2 ; python_version == '2.7'
 future>=0.16,<=0.99 ; python_version == '2.7'
 futures==3.1.1 ; python_version == '2.7'
-jsonref>=0.1.0,<=0.1.99
+jsonref>=0.1,<=0.2
 requests>=2.12.0,==2.*
 jsonschema>=2.5.1,==2.*
 functools32>=3.2,<=3.99 ; python_version == '2.7'
 six>=1.10,<=1.99
 joblib~=0.11
-pubnub~=4
+pubnub~=4.0
```

While you're fixing this, I think you should change it to match the requirements in the `setup.py` file. Could you include a mention of this in the changelog?
```suggestion
pubnub>=4.0,<=4.99
```